Resources/Prototypes/Nyanotrasen/Recipes/Reactions/food.yml

```diff
@@ -0,0 +1,32 @@
+# Curd Cheese
+- type: reaction
+  id: CurdlingCurdCheese
+  impact: Low
+  quantized: true
+  reactants:
+    Milk:
+      amount: 15
+    Vinegar:
+      amount: 5
+    Cream:
+      amount: 5
+  effects:
+    - !type:CreateEntityReactionEffect
+      entity: FoodCurdCheese
```

Comment:
20u worth of ingredients, but only 5u of food that comes out? That's immensely lossy, even for curdling.

Reply:
Hmm you're right. Base CheeseWheel is 40u of ingredients for 15u of Food
So I could divide the ingredients requirement for Curd and Mozzarella so it roughly does 20u ingredients for 10u food